wasm/examples/nodejs/javascript/general/derivation.js

```diff
@@ -0,0 +1,67 @@
+const kaspa = require('../../../../nodejs/kaspa');
+const {
+    Mnemonic,
+    XPrv,
+    DerivationPath,
+    PublicKey,
+    NetworkType,
+} = kaspa;
+
+kaspa.initConsolePanicHook();
+
+(async () => {
+
+    //const mnemonic = Mnemonic.random();
+    const mnemonic = new Mnemonic("hunt bitter praise lift buyer topic crane leopard uniform network inquiry over grain pass match crush marine strike doll relax fortune trumpet sunny silk")
+    console.log("mnemonic:", mnemonic);
+    const seed = mnemonic.toSeed();
+    console.log("seed:", seed);
+
+    // kaspa
+    let xPrv = new XPrv(seed);
+    // derive full path upto second address of receive wallet
+    let pubkey1 = xPrv.derivePath("m/44'/111111'/0'/0/1").toXPub().publicKey();
```

Comment:
```suggestion
    let pubkey1 = xPrv.derivePath("m/44'/111111'/0'/0/1").toXPub().toPublicKey();
```

Reply:
Fixed in https://github.com/aspectron/rusty-kaspa/commit/a0d677aa7f581d13324baba4df4eaddd2caefc21